Build/exec.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+#Use lower case to specify the CPU type ("v2l" or "g2l")
+TYPE="v2l"
```

Comment:
We are in the `rzv2l` repository. This variable is not needed.

Reply:
Yes, it's used on lines 48,49 & 50 to create the file names

Reply:
and line 44

Reply:
I would rather have those file names hardcoded instead of created.

Reply:
I would rather have those file names hardcoded instead of created.

Reply:
mmh - my throught process was to make the change from V2L to G2L as easy as possible but I can see what you're saying too and it makes sense, too looking at it from a different angle!

Reply:
assigned download URLs to variables for change on top of the file, thoughts?